common/test/acceptance/tests/test_cohorted_courseware.py

```diff
@@ -177,31 +252,98 @@ def login_and_verify_visible_problems(username, email, expected_problems):
         )
 
         login_and_verify_visible_problems(
+            self.cohort_verified_student_username, self.cohort_verified_student_email,
+            [self.group_verified_problem, self.visible_to_all_problem],
+            'verified'
+        )
+
+        login_and_verify_visible_problems(
+            self.cohort_audit_student_username, self.cohort_audit_student_email,
+            [self.group_audit_problem, self.visible_to_all_problem],
```

Comment:
I'm confused why cohort_a_student_username and cohort_b_student_username don't see the group_audit_problem (lines 246 and 251), as they should be in the default course mode.